src/providers/useRequestProcessor.js

```diff
@@ -0,0 +1,61 @@
+import { useCallback } from 'react'
```

Comment:
Not sure, but this looks like it shouldn't be in the `providers` folder?

Reply:
Ah shit, i miss understood one comment above, thought that initially you were suggesting to move it to the providers folder since it uses a lot of the providers hooks https://github.com/aragon/court-dashboard/pull/335#discussion_r428560449 but i can see you were referring to the activity provider haha.